wiki/FAQ/en.md

```diff
@@ -56,161 +77,225 @@ _[Osu! Tatakae! Ouendan][ouendan wikipedia]_ is a rhythm game for the Nintendo D
 
-osu!cursor placement does not matter when playing.
-```
+It does not increment the combo counter, and does not recover the healthbar in any way. Each successful denden hit only gives a static 300 score, and denden completion will give a perfect(GREAT) large note score.
 
-You may configure the keyboard buttons as you see fit, from the Input tab in the Options menu.
+#### Controls
+The default controls for the osu!taiko are:-
 
-### What is "osu!catch"?
+Type | Mouse | Keyboard | TaTaCon
+---|---|---|---
+Red |Left click|`X` / `C`|Flat surface of the drum
+Blue|Right click|`Z` / `V`|Outer surface of the drum
+
+The placement of in-game cursor does not matter when playing.
+
+### What is osu!catch?
 _Full explanation: [osu!catch][osu!catch wikilink]_
 
 ![osu!catch Interface][osu!catch Interface image]
 
-_osu!catch_ (previously known as _Catch the Beat_) is the second special game mode to be added in, after osu!taiko game mode.
+_osu!catch_ (previously known as _Catch the Beat_) is the second game mode added to osu!, after osu!taiko game mode.
 
 Instead of clicking circles, you are now tasked to move the catcher at the bottom of the screen while the fruits fall from above.
 
+#### Basic explanation
+All fruits will stay at the plate until its combo ends, which the fruits will bounce out of the plate. Timing and where it falls on the plate does not matter; either a full score for successful catch or miss.  
+
+##### Fruits
+![Fruits][Fruits image]
+
+Fruits are the hitcircles of osu!catch, and it rains down from top of the screen. To catch these fruits, make sure it falls to the plate, and not over the plate. Collected fruits will always give 300pt each, increment combo by 1, and treated as 300 in result screen.
+
+##### Fruit trails
+![Fruit trails][Fruit trails image]
+
+Fruit trails are the sliders of osu!catch. A typical fruit trail will contains two fruits (start and end), droplets (sliderpath), juice drop (sliderticks), and more fruits for each repeat points along with a reversed sliderpath. Droplet gives 10pt each and treated as 50 in result screen. Juice drop gives 30pt each, increment combo by 1, and treated as 100 in result screen.
+
+Missing the juice drop will break the combo (as it is a slidertick), but missing the droplet will not break the combo. However, missed droplets will not be shown in the local result screen, but will be shown in the website's leaderboard as _Miss Droplet_. Miss Droplet notably affects accuracy (missed out the droplet, which is a 50) and score (10pt for each droplet missed).
+
+##### Bananas
+![Bananas][Bananas image]
+
+Bananas serves as spinners in osu!catch. Here, large bananas will rain down and shrink down to catch size as it falls down to the plate (or not). Each collected bananas will give a static score of 1,100 regardless of mod and combo multiplier.
+
+The bananas can be ignored with no health penalty. However, it will cost possible points gained and health recovery from collecting the bananas.
```

Comment:
Last sentence:
`However, it can cost the extra points and health recovery from collecting the bananas.`